dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/parameters/Parameters.java

```diff
@@ -28,10 +28,10 @@
 
     private static final Logger logger = LoggerFactory.getLogger(Parameters.class);
 
-    public static final String PROCESSOR = "processor";
+    public static final String JOB = "job";
 
-    @Field(PROCESSOR)
-    private String processor;
```

Comment:
I am a bit worried about possible clashes with Spring beans and stuff. Maybe this could be called `jobName`?

Reply:
I don't think jobName has less probability of clash with Spring

Reply:
ImportJobName? dbSnpImportJob? dbSnpImportJobName?